### PR TITLE
Add opreq status

### PR DESCRIPTION
**Pull request by kflorianski-ibm**

### CODE REVIEW

**Review by Jan-Olszowka — APPROVED** [review]

**Review by ljeda — APPROVED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Monitoring
     certified: "false"
-    containerImage: icr.io/cpopen/ibm-licensing-operator:4.2.10
-    createdAt: "2024-09-19T11:09:27Z"
+    containerImage: icr.io/cpopen/ibm-licensing-operator:4.2.9
```
Comment (Jan-Olszowka) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:29]: Do we want to downgrade operator version? I don't think so - 4.2.9 was already released.

**Review by Jan-Olszowka — CHANGES_REQUESTED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (kflorianski-ibm) [reply bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:29]: Ehhh, some merge-conflict resolution went wrong

**Review by Jan-Olszowka — APPROVED** [review]

**Review by ljeda — APPROVED** [review]

**Review by Pejdzor — APPROVED** [review]
/lgtm

**Pull request merged**